packages/plugin/src/Services/FormsService.php

```diff
@@ -124,6 +125,16 @@ public function getAllFormIds(): array
         ;
     }
 
+    public function getAllRegularFormIds(): array
+    {
+        return $this
+            ->getFormQuery()
+            ->select('id')
+            ->where(['type' => Regular::class])
+            ->column()
+        ;
+    }
```

Comment:
I would modify this to broaden the use case.
Expose the method for fetching form ID's but let the dev specify a `$type` if they want, which will then modify the query to filter out a specific type.

Then in the place where you use this, pass in the `Regular::class` as the function argument.

```suggestion
    public function getAllFormIds(?string $type = null): array
    {
        $query = $this>getFormQuery()>select('id');
        if (null !== $type) {
            $query->where(['type' => $type]);
        }
        
        return $query->column();
    }
```